File: frontend/src/components/field-labels.ts
export const CORE_FIELD_LABELS_ZH: Record<string, string> = {
  feature_tree: "功能树",
  strengths: "优势",
  weaknesses: "劣势",
  pricing_model: "定价模式",
  user_feedback: "用户反馈",
};

export function fieldLabelZh(fieldName: string): string {
  const key = String(fieldName || "").trim();
  if (!key) return "";
  if (CORE_FIELD_LABELS_ZH[key]) return CORE_FIELD_LABELS_ZH[key];
  return key.replace(/_/g, " ");
}

File: backend/app/core/field_labels.py
from __future__ import annotations


CORE_FIELD_LABELS_ZH: dict[str, str] = {
    'feature_tree': '功能树',
    'strengths': '优势',
    'weaknesses': '劣势',
    'pricing_model': '定价模式',
    'user_feedback': '用户反馈',
}


def field_label_zh(field_name: str) -> str:
    key = str(field_name or '').strip()
    if not key:
        return ''
    if key in CORE_FIELD_LABELS_ZH:
        return CORE_FIELD_LABELS_ZH[key]
    return key.replace('_', ' ')
